FirstLaba.cpp

```diff
@@ -0,0 +1,59 @@
+﻿#include <iostream>
+#include <conio.h>
+
+using namespace std;
+
+int main()
+
+{
+	float first_oper, second_oper;
+	char znak, ch;
+	setlocale(LC_ALL, "Rus");
+	do {
+		cout << "Введите выражение: " << endl;
+		cin >> first_oper;
+		cin >> second_oper;
+		if (cin.good() == true) {
+			cin >> znak;
+			if (znak != '+' && znak != '-' && znak != '/' && znak != '*') {
+				cout << "error! write the correct operator! Press Enter" << endl;
+			}
+			else {
+				switch (znak) {
+				case '+': {
+					cout << first_oper + second_oper << endl;

```

Comment:
Extract `cout << first_oper + second_oper << endl;` from switch..case in order to reduce code duplication